include/common-types.h

```diff
@@ -0,0 +1,25 @@
+#ifndef CONNECT_SDK_COMMON_TYPES_H
+#define CONNECT_SDK_COMMON_TYPES_H
+
+#include <string>
+#include <vector>
+#include <map>
+#include <boost/move/unique_ptr.hpp>
+#include <boost/chrono/system_clocks.hpp>
+
+namespace prism {
+namespace connect {
+
+using std::string;
+using std::vector;
+using std::map;
+
+using boost::movelib::unique_ptr;
```

Comment:
I do not like "using type;" in headers. 
it becomes a problem later.
it is typically allowed only in *.cpp files.
In headers, I prefer typedef for non templates or
type<T>::t kind of typedef for templates as we had in cpp engine. 
It is still pretty compact and no issues later.